### ISSUE TITLE
redis running status can't be got, redis will try to start all the time

### ISSUE THREAD

**Issue by faith2god** [ochinchina/supervisord#208]
it occurs on supervisor v0.5 and v0.6
ubuntu 14.04
it works fine on python  supervisor
it seems like supervisord don't get the status of redis normally.
```
[44965] 18 May 18:37:10.447 # Creating Server TCP listening socket 192.168.1.6:6380: bind: Address already in use
[44967] 18 May 18:37:10.550 * Max number of open files set to 10032
[44967] 18 May 18:37:10.551 # Creating Server TCP listening socket 192.168.1.6:6380: bind: Address already in use
[44969] 18 May 18:37:10.654 * Max number of open files set to 10032
[44969] 18 May 18:37:10.654 # Creating Server TCP listening socket 192.168.1.6:6380: bind: Address already in use
[44971] 18 May 18:37:15.759 * Max number of open files set to 10032
[44971] 18 May 18:37:15.774 # Creating Server TCP listening socket 192.168.1.6:6380: bind: Address already in use
```

**Comment by ochinchina** [comment]
@faith2god Can you provide your supervisor configuration file? And Have you tested the latest supervisor?

**Comment by faith2god** [comment]
I have solved this problem on supversior v0.6.3
the reason is "daemonize yes" in my redis config file
and supervisor need manager this progrem by "daemonize no"
